.github/workflows/release-app.yml

```diff
@@ -52,26 +52,36 @@ jobs:
           CLIENT_ID: ${{ secrets.VAAS_CLIENT_ID }}
           CLIENT_SECRET: ${{ secrets.VAAS_CLIENT_SECRET }}
         run: |
-          composer install --quiet
+          composer install
           ./vendor/bin/phpunit --bootstrap tests/unittests/bootstrap.php tests/unittests/ --testdox
 
       - name: install nextcloud
         env:
           CLIENT_ID: ${{ secrets.VAAS_CLIENT_ID }}
           CLIENT_SECRET: ${{ secrets.VAAS_CLIENT_SECRET }}
-        run: ./install.sh ${{ matrix.nextcloud_version }}
+        run: ./install.sh ${{ matrix.nextcloud_version }} 1
 
       - name: run tests
+        id: bats-tests
         env:
           CLIENT_ID: ${{ secrets.VAAS_CLIENT_ID }}
           CLIENT_SECRET: ${{ secrets.VAAS_CLIENT_SECRET }}
-        run: bats --no-parallelize-across-files --jobs 2 ./tests/bats
+        run: |
```

Comment:
This If was implemented so the artifact upload can still be done so we can investigate stuff like the coredump error.